svn/trunk/html/embed.php

```diff
@@ -12,6 +12,27 @@
 $var = $GLOBALS['HYVOR_TALK_PLUGIN_JS_CONFIG'];
 ?>
 
+<?php if ($var['loadMode'] === 'click'): ?>
+
+    <div style="text-align:center">
+        <button id="hyvor-talk-load-button" onclick="this.style.display='none'">
+            <?php echo __('Load Comments') ?>
+        </button>
+    </div>
+
+    <script>
+        document.getElementById("hyvor-talk-load-button").addEventListener("click", function() {
+            const hyvorTalkComments = document.querySelector("hyvor-talk-comments");
+            hyvorTalkComments.load();
+        });
```

Comment:
A feature in v3 is that users can add multiple comments sections using the shortcode in the page.

So, using the id `"hyvor-talk-load-button"` would clash with each other if you have multiple elements.
Same with selecting `document.querySelector("hyvor-talk-comments");`

I believe the best option would be to generate a random unique string, and add it to the IDs and the `<hyvor-talk-comments>` element as a data attribute and select using them